### ISSUE TITLE
schema validation errors in issues.severity that should be valid

### ISSUE THREAD

**Issue by ryancurrah** [golangci/golangci-lint#4616]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've read the typecheck section of the FAQ (https://golangci-lint.run/usage/faq/#why-do-you-have-typecheck-errors).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

Schema shows not valid for `issues.severity`. Though looking at the code it seems my config is valid. I think something is wrong with the schema.

### Version of golangci-lint

<details>

```console
❯ golangci-lint --version
golangci-lint has version 1.57.2 built with go1.22.1 from 77a8601 on 2024-03-28T18:39:06Z
```

</details>

### Configuration

<details>

```console
run:
  modules-download-mode: readonly

severity:
  default-severity: error
  case-sensitive: false
  rules:
    - severity: info
      path: _test\.go # Lint issues in tests should never block a pull request
    - severity: info
      linters:
        - deadcode
        - funlen
        - gochecknoinits
        - gocognit
        - goconst
        - gofmt
        - goimports
        - lll
        - misspell
        - shadow
        - structcheck
        - unconvert
        - unparam
        - unused
        - varcheck
    - severity: info
      linters:
        - revive
      text: "package-comments:"
    - severity: info
      linters:
        - revive
      text: "exported:"
    - severity: info
      linters:
        - revive
      text: "var-naming:"
    - severity: info
      linters:
        - gocritic
      text: "rewrite if-else to switch statement"
    - severity: info
      linters:
        - staticcheck
      text: "SA1019" # Allow using a deprecated function, variable, constant or field
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
❯ golangci-lint config verify
jsonschema: "severity.rules.2" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/0/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.2" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/1/$ref/additionalProperties": additionalProperties 'text', 'linters' not allowed
jsonschema: "severity.rules.2" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/2/$ref/additionalProperties": additionalProperties 'text' not allowed
jsonschema: "severity.rules.2" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/3/$ref/additionalProperties": additionalProperties 'linters' not allowed
jsonschema: "severity.rules.3" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/0/$ref/additionalProperties": additionalProperties 'text', 'linters' not allowed
jsonschema: "severity.rules.3" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/1/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.3" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/2/$ref/additionalProperties": additionalProperties 'text' not allowed
jsonschema: "severity.rules.3" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/3/$ref/additionalProperties": additionalProperties 'linters' not allowed
jsonschema: "severity.rules.4" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/0/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.4" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/1/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.4" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/2/$ref/additionalProperties": additionalProperties 'text' not allowed
jsonschema: "severity.rules.4" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/3/$ref/additionalProperties": additionalProperties 'linters' not allowed
jsonschema: "severity.rules.5" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/0/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.5" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/1/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.5" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/2/$ref/additionalProperties": additionalProperties 'text' not allowed
jsonschema: "severity.rules.5" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/3/$ref/additionalProperties": additionalProperties 'linters' not allowed
jsonschema: "severity.rules.6" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/0/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.6" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/1/$ref/additionalProperties": additionalProperties 'linters', 'text' not allowed
jsonschema: "severity.rules.6" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/2/$ref/additionalProperties": additionalProperties 'text' not allowed
jsonschema: "severity.rules.6" does not validate with "/properties/severity/properties/rules/items/allOf/0/oneOf/3/$ref/additionalProperties": additionalProperties 'linters' not allowed
Error: the configuration contains invalid elements
Usage:
  golangci-lint config verify [flags]

Global Flags:
      --color string   Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -c, --config PATH    Read config from file path PATH
  -h, --help           Help for a command
      --no-config      Don't read config file
  -v, --verbose        Verbose output

Failed executing command with error: the configuration contains invalid elements
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [X] Yes, I've included all information above (version, config, etc.).

**Comment by ryancurrah** [comment]
Thank you for the quick fix!
